cfme/tests/test_login.py

```diff
@@ -51,3 +52,5 @@ def test_bad_password(context, request, appliance):
     with appliance.context.use(context):
         with error.expected(error_message):
             appliance.server.login(user)
+        view = appliance.browser.create_view(LoginPage)
```

Comment:
**Question**: Will this issue be fixed in 5.8? If not, please add version check here not to fail 5.8 test

Reply:
@rlbabyuk Yes sure thanks. I am adding version check.